infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -343,8 +343,13 @@ private void bootstrapLogReplicationService() {
 
         logReplicationServerHandler = new LogReplicationServer(serverContext, logReplicationConfig,
             logReplicationMetadataManager, localCorfuEndpoint, topologyDescriptor.getTopologyConfigId(), localNodeId);
-        logReplicationServerHandler.setActive(localClusterDescriptor.getRole().equals(ClusterRole.ACTIVE));
-        logReplicationServerHandler.setStandby(localClusterDescriptor.getRole().equals(ClusterRole.STANDBY));
+        if (localClusterDescriptor.getRole().equals(ClusterRole.ACTIVE)) {
+            logReplicationServerHandler.setActive(true);
+            addDefaultReplicationStatus();
+        } else if (localClusterDescriptor.getRole().equals(ClusterRole.STANDBY)) {
+            logReplicationServerHandler.setStandby(true);
+            statusFlag = false;
```

Comment:
Another thing to note is that this initialization must be done on a role change from Standby -> Active and also when new standby clusters get added.  This method may not get invoked every time.